docs/getting-started/quickstart.md

```
@@ -39,9 +39,36 @@ data = pd.DataFrame({
+result = enrich(
+    "products.csv",
+    "Extract the brand name from: {product}",
+    output_columns=["brand"],
+    model="gpt-4o-mini",
+)
+
+print(result)
+```
+
+**Output:**
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)